### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by varholak-peter**
graph be done (sorta)

### CODE REVIEW

File: src/api/fetch.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const dummyPatients = [
 ];
 
 const fetchData = (url, callback, temporaryFakeValue = dummyPatients) => {
-  console.log(`Fetching: ${ ENDPOINT }${ url }`);
+  console.log(`Fetching: ${ ENDPOINT }${ url }`); // eslint-disable-line
```
Comment (varholak-peter) [inline src/api/fetch.js]: removes console polution

File: src/components/chart/chart.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  },
+  +options: Object, // TODO: change later
+  +getRef: Function,
+}
+
+const maxData = 800;
+const maximum = Math.log10(maxData);
+const dummyData = Array(maxData).fill().map((_, i) => (
+  Math.log10(i) / maximum
+  // (Math.random() * 0.2)
```
Comment (garySiladi) [inline src/components/chart/chart.js]: console?

File: src/components/chart/chart.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        {
+          data: dummyData,
+          label: 'Median',
+          borderColor: '#9ccef9',
+          backgroundColor: this.state.medianBackground,
+          pointStyle: 'line',
+          showLines: false,
+        }, {
+          data: dummyData.map(entry => (entry + (Math.random() * 0.2)) - 0.1),
+          label: 'Cancur',
```
Comment (garySiladi) [inline src/components/chart/chart.js]: :D I literally laughed out loud when I saw this

File: src/components/chart/chart.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+          borderWidth: 2,
+          fill: false,
+          pointStyle: 'dash',
+        },
+      ],
+    };
+    const options = {
+      title: {
+        display: true,
+        text: 'You got cancur',
```
Comment (garySiladi) [inline src/components/chart/chart.js]: 😄

File: src/app/app.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ class App extends React.Component<{}> { // eslint-disable-line react/prefer-stat
           <div>
             <Header />
             <Route exact path="/" component={Homepage} />
-            <Route path="/patient" component={PatientInfo} />
+            <Route path="/patient" component={Graph} />
```
Comment (garySiladi) [inline src/app/app.js]: I think I would add a new route, not replace ... cause we want to build on the patient layout

File: src/components/chart/chart.js (hunk 2 above)
Comment (varholak-peter) [reply src/components/chart/chart.js]: will remove

File: src/app/app.js (hunk 5 above)
Comment (varholak-peter) [reply src/app/app.js]: it doesn't really matter now until we have that screen

Comment (garySiladi) [reply src/app/app.js]: I'm saying you should create that new screen, this replacement creates just conflict

**Pull request merged**